.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+<!--
```

Comment:
> You are using an old version of issue templates. Please update to the new issue template workflow.

It seems we are using an old template version. You can check the warning when watching the file: https://github.com/asyncapi/spec/blob/3535538236c899d282847aa979425acb0e54567e/.github/ISSUE_TEMPLATE.md

Reply:
Good catch. Thanks, Sergio.

Reply:
I added a frontmatter but it's still complaining. Not sure what to do. It doesn't say anything on the referenced docs 🤷

Reply:
You can create them via https://github.com/asyncapi/spec/issues/templates/edit

Reply:
Yeah, that doesn't help. It allows you to create templates for different types of issues. What I'm editing here is the default one when people choose "Open new blank issue".